content/hello-world-program.md

```diff
@@ -0,0 +1,328 @@
+# Hello World
+
+# Lesson Objectives
+
+*By the end of this lesson, you will be able to:*
+
+- Describe the Solana account model
+- Explain the entry point to a Solana program
+- Submit a transaction to invoke our “Hello, world!” program
+
+# TL;DR
+
+- **Accounts** on Solana each have a unique address, store data, and are owned by a program
+- **Programs** on Solana are a particular type of account that is marked as executable and which stores instruction logic
+- **Transactions** on Solana are made up of instructions which are executed by programs
+
+# Overview
+
+In this lesson we'll be using the Rust programming language to write a Solana program. We will also be using the Solana Playground to deploy a Solana program.
+
+Solana programs are stored in accounts with a unique address. This unique address is referred to as the program id.
+
+To understand programs on Solana we must first understand the basics of Solana’s account model. Once we understand Solana's account model, then we will be able to see how Solana programs (which are similar to "smart contracts" on the Ethereum blockchain) are just a particular type of Solana account that can execute instructions.
+
+## Account Model
```

Comment:
I don't think we should go over the entire Accounts model. Even if we do, this title should be targeted to Programs so that readers feel like it's connected to everything else. So maybe something like "What are Solana Programs"

Reply:
removed account model section